test/TestApps/ToastNotificationsTestApp/ToastNotificationsTestApp.cpp

```diff
@@ -237,6 +251,12 @@ int main() try
 
     ::Test::Bootstrap::SetupBootstrap();
 
+    if (Test::AppModel::IsPackagedProcess())
```

Comment:
Why registering at this point, rather than in the test? Is it related to make background activation work?